### PR TITLE
Fix type static string and make const

### PR DESCRIPTION
**Pull request by mehdi-fathi**
Fix type static string and make const and change name of var for make better code.

### CODE REVIEW

File: src/View/View.php [hunk 1]
```diff
@@ -197,7 +197,7 @@ class View implements EventDispatcherInterface
      *
      * @var array
      */
-    public $uuids = [];
+    public $uuIds = [];
```
Comment (markstory) [inline src/View/View.php]: This is a breaking change. You cannot rename public properties.

File: src/View/View.php [hunk 2]
```diff
@@ -830,7 +837,7 @@ public function extend($name)
                     if (!$parent) {
                         list($plugin, $name) = $this->pluginSplit($name);
                         $paths = $this->_paths($plugin);
-                        $defaultPath = $paths[0] . 'Element' . DIRECTORY_SEPARATOR;
+                        $defaultPath = $paths[0] . ucfirst(static::TYPE_ELEMENT) . DIRECTORY_SEPARATOR;
```
Comment (markstory) [inline src/View/View.php]: Adding a function call adds overhead. Why not add a constant with the appropriate value?

Comment (markstory) [inline src/View/View.php]: Perhaps `NAME_ELEMENT`?

**Review by markstory — CHANGES_REQUESTED** [review]
You have breaking changes right now. The public property change needs to be removed for this to be mergeable.

File: src/View/View.php (hunk 2 above)
Comment (mehdi-fathi) [inline src/View/View.php]: Yes NAME_ELEMENT is better than

File: src/View/View.php [hunk 3]
```diff
@@ -1,16 +1,16 @@
 <?php
 /**
- * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
- * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
```
Comment (ADmad) [inline src/View/View.php]: Undo these http to https changes. This header needs to stay consistent across all files.

You can make a separate PR to change it for all files if you want.

**Review by ADmad — CHANGES_REQUESTED** [review]

File: src/View/View.php (hunk 3 above)
Comment (mehdi-fathi) [inline src/View/View.php]: Yes i think a separate commit is good for this work.Because of almost all of file's url cakephp.org has this problem.

**Comment by markstory** [comment]
Do you have a scenario where you needed to change the constants?

**Comment by mehdi-fathi** [comment]
@markstory Actually i don't have.But it better than former code.When you use a string can will be use more than one time.Why it be not a constants.

**Comment by markstory** [comment]
I wasn't arguing for not having constants. Just wondering what your usecase/motivation was to change this :smile:

**Review by markstory — APPROVED** [review]

**Comment by mehdi-fathi** [comment]
Nothing.just a name of element constant for access another place.maybe there is no need.

**Pull request merged**